tests/Fuzzed.nim

```diff
@@ -1,29 +1,52 @@
 import unittest
```

Comment:
If every file now needs to import Fuzzed for noFuzzTest, we should export unittest so we don't we need to import both.

Reply:
To append to this comment, we should be able to explicitly export the suite macro, which also resolves a below comment about stopping accidental `test` usage.